tests/test_core/test_sequence.py

```diff
@@ -2566,3 +2566,29 @@ def test_gapped_by_map_segment_iter():
     m, seq = moltype.make_seq("-TCC--AG").parse_out_gaps()
     g = list(seq.gapped_by_map_segment_iter(m, allow_gaps=True, recode_gaps=False))
     print(g)
+
+
+@pytest.mark.parametrize("rev", (False, True))
+@pytest.mark.parametrize("sliced", (False, True))
+@pytest.mark.parametrize("start_stop", ((None, None), (3, 7)))
+def test_copied_parent_coordinates(sliced, rev, start_stop):
+    seq = DNA.make_seq("ACGGTGGGAC")
+    start, stop = start_stop
+    start = start or 0
+    stop = stop or len(seq)
+    sl = slice(start, stop)
+    seq = seq[sl]
+    if rev:
+        seq = seq.rc()
+    copied = seq.copy(sliced=sliced)
+    assert copied.parent_coordinates() == (start, stop)
+
```

Comment:
additionally could test that a seq that has been sliced, then copied with `sliced=False`, will have the same start and stop in the copied as in the original.